src/player.js

```diff
@@ -202,6 +202,14 @@ const MPRISPlayer = new Lang.Class({
           if (props.CanGoPrevious)
             newState.canGoPrevious = props.CanGoPrevious.unpack();
 
+          if (props.CanSeek) {
+            newState.canSeek = props.CanSeek.unpack();
+            if (this.state.canSeek !== newState.canSeek) {
+              this.emit('player-update', newState);
```

Comment:
newState is already emited at the end of the function. I think you should first check if canSeek has changed. If yes, add it to the newState object and run _getPosition, if not do nothing.